backend/src/main/java/com/bakdata/conquery/models/query/QueryExecutor.java

```diff
@@ -65,4 +68,9 @@ public void close() throws IOException {
 			throw new RuntimeException(e);
 		}
 	}
+	
+	public boolean isBusy () {
+		// This might not be super accurate (see the Documentation of ThreadPoolExecutor)
+		return executor.getActiveCount() != 0 || !executor.getQueue().isEmpty();
```

Comment:
wobei `!= 0` afaik ja immer halten sollte wenn etwas läuft?

Reply:
ActiveCount kann null sein, wenn gerade nichts bearbeitet wird aber noch Jobs in der Queue sind, z.B.  durch einen Kontextwechsel